File: code/ide/adt/L03_06UsingAdapter/src/cn/eoe/usingadapter/People.java
package cn.eoe.usingadapter;

public class People {

	public People(String name,int age) {
		this.name = name;
		this.age = age;
	}
	
	public String getName() {
		return name;
	}
	
	public int getAge() {
		return age;
	}
	
	private String name;
	private int age;
}

File: code/ide/eclipse/L11_02JS/WebContent/cn/eoe/People.js
//if (!window.cn) {
//window.cn = {};
//}
//if (!window.cn.eoe) {
//window.cn.eoe = {};
//}

//window.cn = window.cn||{};
//cn.eoe = {};

(function(){

	/**
	 * @param {String} name
	 */
	cn.eoe.People = function(name){

		this._name = name;

		this.sayHello = function(){
			console.log("Hello "+this._name);
		};
	};
}());